basic_simulation_vehicle/README.md

```diff
@@ -0,0 +1,4 @@
+# Basic Simulation Vehicle CARMA Config
+This is a configuration image for basic carma vehicles in simulation. It only includes `/opt/carma/vehicle/config/` files and not CARMA Platform plugin configuration files since CARMA Config does not currently handle plugin configuration files. The `docker-compose.yml` defined in this directory includes bridge images for CARMA Platform integration into CDASim including
+- NS3 Adapter
+- Carma Carla Integration
```

Comment:
In general I like that the configuration management is getting attention. Before we do any changes, it might be good to have a discussion to make sure we account for all usages and edge cases. At least having a high level document would help. That is the case if there is no immediate business need for this change and if this is mostly just an initiative towards our goal. 

That being said, I was literally thinking about this yesterday that we can figure out a better config management pretty quickly and implement it quickly. I don't intend to hold this back for long, so we can discuss this next week, create a quick high level document, and make the overhaul for all folders.

Reply:
Not sure I understand. While I think we can have a bigger conversation about configuration management for CARMA Platform, I do not think this story or the CDASim project currently plans to take that on. What this story aims to do is better us our existing configuration management tools as designed to better facilitate the development of new scenario management for CDASim. With this in mind if we believe there is some way carma-config is being used which is different from its designed purpose I think we can make sure we are not breaking anything by thinking through edge cases. However this seems to me to align exactly with the designed purpose and so I am not sure I see the concern here.

Reply:
Discussion went to offline chat